[release-0.97] unit-test CI lane broken over monitoringlinter

**What happened**:
lane [fails](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_cluster-network-addons-operator/2004/pull-cluster-network-addons-operator-unit-test-release-0.97/1891115450314526720):
```
+ make lint-monitoring
GOBIN=$(pwd)/build/_output/bin/ /tmp/cnao/cluster-network-addons-operator/build/_output//bin//go//bin//go install -mod=mod github.com/kubevirt/monitoring/monitoringlinter/cmd/monitoringlinter@e2be790
go: downloading github.com/kubevirt/monitoring v0.0.5
go: downloading github.com/kubevirt/monitoring/monitoringlinter v0.0.0-20240312134250-e2be7906e760
go: downloading golang.org/x/tools v0.18.0
go: downloading golang.org/x/mod v0.15.0
/tmp/cnao/cluster-network-addons-operator/build/_output//bin//monitoringlinter ./...
monitoringlinter: internal error: package "bytes" without types was imported from "github.com/kubevirt/cluster-network-addons-operator/test/kubectl"
make: *** [Makefile:244: lint-monitoring] Error 1
```

**What you expected to happen**:
A clear and concise description of what you expected to happen.

**How to reproduce it (as minimally and precisely as possible)**:
Steps to reproduce the behavior.

**Additional context**:
Add any other context about the problem here.

**Environment**:
- KubeVirt version (use `virtctl version`): N/A
- Kubernetes version (use `kubectl version`): N/A
- VM or VMI specifications: N/A
- Cloud provider or hardware configuration: N/A
- OS (e.g. from /etc/os-release): N/A
- Kernel (e.g. `uname -a`): N/A
- Install tools: N/A
- Others: N/A

--- a/Makefile
+++ b/Makefile
@@ -240,7 +240,7 @@ lint-metrics:
 	./hack/prom_metric_linter.sh --operator-name="kubevirt" --sub-operator-name="cnao"
 
 lint-monitoring:
-	GOBIN=$$(pwd)/build/_output/bin/ $(GO) install -mod=mod github.com/kubevirt/monitoring/monitoringlinter/cmd/monitoringlinter@e2be790
+	GOBIN=$$(pwd)/build/_output/bin/ $(GO) install -mod=mod github.com/kubevirt/monitoring/monitoringlinter/cmd/monitoringlinter@v0.0.8
 	$(MONITORING_LINTER) ./...
 
 clean:

